### ISSUE TITLE
Validation error when editing a case

### ISSUE THREAD

**Issue by jgabrielsantos** [salesagility/SuiteCRM-Core#498]
### Issue

I've been running a bitnami docker image OS: Debian 12.5 for the latest suiteCRM version (>8) and when I try to update a case information, the multiselect required fields throw a validation error as if the field did not have any information, even though they are as when created the record.

Screenshots below
![Screenshot 2024-06-04 at 10 14 15 AM](https://github.com/salesagility/SuiteCRM-Core/assets/48341861/628d2afe-66ec-4a92-890b-c060814dcb83)

### Steps to Reproduce the Issue

```bash
1. Run bitnami docker image
2. Create a new case with a multi select required field
3. Edit the created case
4. Save without making any changes
...
```

### Version

8.5.1

### What browser are you currently using?

Chrome

### Browser Version

125.0.6422.112

### Environment Information

MariaDB11.3, PHP 8.1.28

### Operating System and Version

Ubunto 24.04 LTS

**Comment by chris001** [comment]
What is the result when you do the same on the online demo?  Same errors?  Different errors?  No errors?
[demo.suite8ondemand.com](https://suite8demo.suiteondemand.com/) user will pass will

**Comment by jgabrielsantos** [comment]
@chris001 there's no multi select field in the cases create form. I did add a couple to my project and made them required. Can't replicate in the demo because I didn't find any form with a multi select required field

**Comment by jgabrielsantos** [comment]
hi @mattlorimer , I saw that you're a core contributor to suitecrm-core from opencollective and I was hoping you could help with this issue

**Comment by clemente-raposo** [comment]
Hi @jgabrielsantos @chris001 

The multienum component was updated on 8.6.0 and multiple fixes were also applied on 8.6.1.

I've re-tested this and was not able to reproduce (see the following gif). Closing this issue.

Please update to the latest version and if you still have problems consider re-opening.

![required-multienum-demo](https://github.com/salesagility/SuiteCRM-Core/assets/1140849/92685e0e-ce6c-463a-a991-8bf028313f13)

**Comment by jgabrielsantos** [comment]
hi @clemente-raposo , you're right. If when editing a record the user clears and submit the same values in a multiselect they will have no issues when saving the changes. The issue however remains when the user changes anything else other than the multiselect and try to submit the changes - no interaction with the field. then this leads to a validation error even though the mandatory field does have values
